macaroonbakery/httpbakery/error.py

```diff
@@ -78,21 +77,21 @@ def request_version(req_headers):
     version is used, which is OK because versions are backwardly compatible.
 
     @param req_headers: the request headers as a dict.
-    @return: bakery protocol version (for example macaroonbakery.BAKERY_V1)
+    @return: bakery protocol version (for example bakery.BAKERY_V1)
```

Comment:
I think I'd keep the original name in the docstring (eg, macaroonbakery.BAKERY_V1) as I'm not sure the alias is visible when viewing the docs directly.

Reply:
good point. done.